spec/template.dd

```diff
@@ -1394,11 +1394,27 @@ $(GNAME Constraint):
             ...
         }
         ...
-        foo!(3)();  // ok, matches
-        foo!(4)();  // error, no match
+        foo!(3)();  // OK, matches
+        foo!(4)();  // Error, no match
+        ---
+    )
+    
+    $(P Template constraints can be used with aggregate types (structs, classes, unions). 
+        Constraints are effectively used with library module "std.traits":
+    
+        ---
+        import std.traits;
+        
+        struct Bar(T)
+            if( isIntegral!T )
```

Comment:
Sorry, this isn't in the style guide yet:

https://github.com/D-Programming-Language/dlang.org/pull/1191

Essentially we place the spaces outside the parens, not inside.

The same style is used in other examples on this page, so it would be nice to avoid inconsistent code style.